tests/gtests/test_iface_wino_convolution.cpp

```diff
@@ -61,6 +61,14 @@ class wino_conv_test_t : public ::testing::Test {
         input_int8.wino_supported = is_cpu && isa >= cpu_isa::avx512_core
                 && isa != cpu_isa::avx2_vnni;
         input_f32.backward_supported = is_cpu && impl::dnnl_thr_syncable();
+#else
+#if DNNL_AARCH64 && DNNL_AARCH64_USE_ACL
```

Comment:
Could you please move this whole `#if` condition after `#endif // DNNL_X64` to separate them?